libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -370,14 +387,23 @@ impl<'a> JsiiImporter<'a> {
 		}
 	}
 
+	fn is_fqn_construct_base(fqn: &str) -> bool {
+		// We treat both CONSTRUCT_BASE and WINGSDK_RESOURCE, as base constructs because in wingsdk we currently have stuff directly derived
+		// from `construct.Construct` and stuff derived `cloud.Resource` (which itself is derived from `constructs.Construct`).
+		// But since we don't support interfaces yet we can't import `cloud.Resource` so we just treat it as a base class.
+		// I'm also not sure we should ever import `cloud.Resource` because we might want to keep its internals hidden to the user:
```

Comment:
```suggestion
		// But since we don't support interfaces yet we can't import `core.Resource` so we just treat it as a base class.
		// I'm also not sure we should ever import `core.Resource` because we might want to keep its internals hidden to the user:
```